README.md

```diff
@@ -115,6 +115,59 @@ Included in this repository is a demo of using Mobile UI Library to start a call
 
 This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments. Also, please check our [Contribution Policy](CONTRIBUTING.md). 
 
+## Using a Custom Library version in your Application
```

Comment:
I would suggest to place this in a separate file and add link to it from this file.